rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -147,10 +162,23 @@ private RMProxyUserInterface checkAccess(String sessionId) throws NotConnectedEx
     public String rmConnect(@FormParam("username") String username, @FormParam("password") String password)
             throws KeyException, LoginException, RMException, ActiveObjectCreationException, NodeException {
 
-        Session session = sessionStore.create(username);
+        if ((username == null || username.trim().isEmpty()) || (password == null || password.trim().isEmpty())) {
```

Comment:
You could use here instead the Apache Commons `StringUtils#isBlank`.

Reply:
I agree, it is better to use StringUtils. Done :)